config/pipelines/high_throughput_custom_pool.yml

```diff
@@ -4,5 +4,6 @@ Custom Pool:
     request_type_key: limber_lcmb
     state: passed
   relationships:
+    Library Stock Plate: LB Custom Pool #new plate purpose for multiplexed sequencing submissions, to avoid FC
```

Comment:
I would put the comment as something more like 
```
These are supplied as ready-made libraries and enter the pipeline at this point.
```

I think labelling it as 'new' will become outdated, and FC needs explaining more.